packages/validator/transpile.ts

```diff
@@ -0,0 +1,8 @@
+import Ajv, { Options as AjvOptions } from 'ajv'
+
+export function transpileSchema (
+  schema: object,
+  ajvOptions?: Partial<AjvOptions>
+): Ajv
+
+export function transpileLocale (src: string, options?: object | any): function
```

Comment:
Shouldn't the return type be `Function`?

Reply:
done, thanks for the review